docs/source/AdministratorGuide/technologyPreviews.rst

```diff
@@ -22,3 +22,21 @@ M2Crypto (or any standard tool that respects TLS..) will be stricter than PyGSI.
 * SAN in your certificates: if you are contacting a machine using its aliases, make sure that all the aliases are in the SubjectAlternativeName (SAN) field of the certificates
 * FQDN in the configuration: SAN normally contains only FQDN, so make sure you use the FQDN in the CS as well (e.g. `mymachine.cern.ch` and not `mymachine`)
 * ComponentInstaller screwed: like any change you do on your hosts, the ComponentInstaller will duplicate the entry. So if you change the CS to put FQDN, the machine will appear twice. 
+
+
+.. _jsonSerialization:
+
+JSON Serialization
+==================
+
+We aim at replacing the DEncode serialization over the network with json serialization. In order to be smooth, the transition needs to happen in several steps:
+* DISET -> DISET: as it is now. We encode DISET, and decode DISET
+* DISET -> DISET, JSON: we send DISET, try to decode with DISET, fallback to JSON if it fails. This step is to make sure all clients will be ready to understand JSON whenever it comes.
+* JSON -> DISET, JSON: we send JSON, attempt to decode with DISET, fallback to JSON if it fails. This step is to make sure that if we still have some old clients lying around, we are still able to understand them.
+* JSON -> JSON: final step, goodbye DISET.
+
+The changes from one stage to the next is controlled by environment variables, and it can go to your own pace:
+* `DIRAC_USE_JSON_DECODE`: must be the first one. Enables the DISET,JSON decoding
```

Comment:
```suggestion

* ``DIRAC_USE_JSON_DECODE``: must be the first one. Enables the DISET,JSON decoding
```

Reply:
I will add this changes when tagging. Any other suggestions ?

Reply:
Thanks! 
Nothing else at the moment